composer.json

```diff
@@ -12,18 +12,15 @@
     "repositories": [
         {
             "type": "vcs",
-            "url": "https://github.com/smmccabe/phpmd.git"
-        },
-        {
-            "type": "vcs",
-            "url": "https://github.com/smmccabe/PHP_CodeSniffer.git"
+            "url": "https://github.com/cobenash/PHP_CodeSniffer.git"
```

Comment:
Let's not start a new fork. Ask @smmccabe to sync changes from upstream

Reply:
corrected in master after merge